lib/ui/src/components/panel/panel.js

```diff
@@ -10,6 +10,20 @@ const DesktopOnlyIconButton = styled(IconButton)({
   },
 });
 
+const tabSorter = (panels, panelOrder) => {
```

Comment:
Could we pull this out into a separate lib file and and a couple of simple tests? (Perhaps coupled with the hidden logic too?)

Reply:
@tmeasday  Sure thing.